src/Confluent.Kafka/Consumer.cs

```diff
@@ -446,8 +447,8 @@ public void Unassign()
         /// <returns>
         ///     Current stored offset or a partition specific error.
         /// </returns>
-        public List<TopicPartitionOffsetError> StoreOffset(Message<TKey, TValue> message)
-            => consumer.StoreOffsets(new[] { new TopicPartitionOffset(message.TopicPartition, message.Offset + 1) });
+        public TopicPartitionOffsetError StoreOffset(Message<TKey, TValue> message)
+            => consumer.StoreOffsets(new[] { new TopicPartitionOffset(message.TopicPartition, message.Offset + 1) })[0];
```

Comment:
Is it safe to assume that it always returns at least one so the [0] deref can be used?

Reply:
yep - if it's not then it'd be an internal error and something to fix somewhere else.